### ISSUE TITLE
Problem with optional parameters

### ISSUE THREAD

**Issue by dastein1** [heftapp/graphql_codegen#160]
I tried to update form version 0.9.0 to 0.10.3 but I'm facing an issue.

```
At run time the Error: Method 'map' cannot be called on 'Iterable<Iterable<Input$FilterOption>>?' because it is potentially null.
package:some-app/some-path/_schema/schema.graphql.dart:1581
 - 'Iterable' is from 'dart:core'.
 ```

_FilterOption_ comes from an optional input type (`filters: [[FilterOption!]!]`)

However the code generated does not make the call conditional.

<img width="803" alt="Screenshot 2022-08-09 at 09 27 19" src="https://user-images.githubusercontent.com/51369447/183590243-50c7450b-c33a-4d48-a231-b0cdc8b34fa3.png">

**Comment by budde377** [comment]
Thanks for reporting this! A fix is released as `0.11.0-beta.4`.

**Comment by dastein1** [comment]
Thanks @budde377 !
